ckanext/canada/cli.py

```diff
@@ -540,35 +566,37 @@ def _capture_exception_details(_reason: str, _package_id: str):
             sys.stdout.flush()
 
 
-def _changed_datasets(since_date, server, brief):
+def _changed_datasets(since_date: str, server: Optional[str],
+                      brief: Optional[bool]):
     """
     Produce a list of dataset ids and requested dates. Each package
     id will appear at most once, showing the activity date closest
     to since_date. Requested dates are preceeded with a "#"
     """
-    since_date = isodate(since_date, None)
+    since_datetime: Optional[datetime] = isodate(since_date, cast(Context, {}))
```

Comment:
why is `since_datetime` optional? If we don't get a datetime wouldn't it be better to exit with an error here?

Reply:
`isodate` can return `None`